galaxy/main/migrations/0083_content_search_vector.py

```diff
@@ -52,7 +52,7 @@ class Migration(migrations.Migration):
         migrations.AddIndex(
             model_name='content',
             index=psql_indexes.GinIndex(
-                fields=[b'search_vector'],
+                fields=['search_vector'],
```

Comment:
this is probably a breaking change

Reply:
Apparently it's not. Checked both on 2 and 3.